vocabularies/voc4cat/0008095.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0008095>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0008095"^^xsd:token ;
+    skos:altLabel "LT"@en ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000186> ;
+    skos:definition "The fraction of the total measurement time during which the detector was live and able to record counts."@en ;
+    skos:historyNote "0009-0004-3786-0773 Carla Terboven created the resource"@en ;
+    skos:inScheme <https://w3id.org/nfdi4cat/voc4cat> ;
+    skos:prefLabel "live time"@en ;
+.
```

Comment:
I wonder if "active time" is a better term (based on the definition). But if "live time" is an established so be it.